lib/chitter.rb

```diff
@@ -0,0 +1,37 @@
+require 'pg'
+
+class Chitter
+
```

Comment:
Although it is clear what this class is doing, could consider naming this the 'Peep' class instead as this model file specifically adds and creates new 'peeps'. Then for other user stories could have a 'User' class, which is responsible for signing in users.